spec/support/spree.rb

```diff
@@ -6,5 +6,5 @@
 
 RSpec.configure do |config|
   config.include Spree::TestingSupport::UrlHelpers
-  config.include Spree::TestingSupport::ControllerRequests
-end
\ No newline at end of file
+  config.include Spree::TestingSupport::ControllerRequests, type: :controller
```

Comment:
Wait, didn't we just set `infer_spec_type_from_file_location!` so that we wouldn't need this?